docs/reference/api/connectivity/nfc/NFCEEPROM.md

```diff
@@ -0,0 +1,28 @@
+## NFC EEPROM
+
+To use an NFC controller, you must initiate the instance with a driver instance, an event queue and a scratch buffer for NDEF messages.
```

Comment:
I'm not sure how this sentence about an NFC controller ties in to this page about NFC EEPROM. Could we say what NFC EEPROM is, what it stands for and when/why you'd want to use it?